DynamicPackagingVODContent/AssetFilters/Program.cs

```diff
@@ -11,17 +11,17 @@
 using Microsoft.Extensions.Configuration;
 using Microsoft.Rest;
 using Microsoft.Rest.Azure.Authentication;
-using Microsoft.WindowsAzure.Storage.Blob;
 using Microsoft.IdentityModel.Clients.ActiveDirectory;
 using System.Text.RegularExpressions;
+using Microsoft.Azure.Storage.Blob;
 
 namespace AssetFilters
 {
     class Program
     {
         private const string adaptiveTransformName = "MyTransformWithAdaptiveStreamingPreset";
         private const string InputMP4FileName = @"ignite.mp4";
-        private const string DefaultStreamingEndpointName = "se";   // Change this to your Streaming Endpoint name.
+        private const string DefaultStreamingEndpointName = "default";   // Change this to your Streaming Endpoint name.
```

Comment:
@xpouyat  - I recall that there was a distinct reason we chose not to use 'default' at one point.  Might want to ask Julia or others on the team if they recall why.

Reply:
When I tried, the app crashed because it did not find the se streaming endpoint. I thin "default" is a better bet. Most of customers have it.